controllers/eventingauth_controller_test.go

```diff
@@ -134,6 +135,15 @@ var _ = Describe("EventingAuth Controller unhappy tests", Serial, Ordered, func(
 		stubSuccessfulSkrSecretCreation()
 		verifyEventingAuthStatusReady(eventingAuth)
 	})
+
+	It("should cleanup if no kyma cr exists", func() {
+		stubSuccessfulIasAppCreation()
+		stubSuccessfulSkrSecretCreation()
+		eventingAuth = createEventingAuth(crName)
```

Comment:
Can we add one more step after `eventingAuth = createEventingAuth(crName)` to verify that it exists?

Reply:
is implicitly covered by the next line:
`	reconcileEventingAuth(eventingAuth)` will only succeed if the EA-CR exists